docs/source/examples/smoothing.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<p>Similarly it would be nice here to have a comparison to a few singular lead years to show that smoothing reduces the error.Could have for instance LY 1,2,3,4 compared to smoothed LY1-4 and subtract them to show differences.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I added this for the PM setup. its basically the same here. I dont see the added value.</p>